scripts/zkai_jstprove_shape_probe.py

```
@@ -0,0 +1,593 @@
+import time
+from typing import Any
+
+
+ROOT = pathlib.Path(__file__).resolve().parents[1]
+JSON_OUT = ROOT / "docs" / "engineering" / "evidence" / "zkai-jstprove-shape-probe-2026-05.json"
+TSV_OUT = ROOT / "docs" / "engineering" / "evidence" / "zkai-jstprove-shape-probe-2026-05.tsv"
+
+SCHEMA = "zkai-jstprove-shape-probe-v1"
+DECISION = "GO_OPERATOR_SUPPORT_SPLIT_NOT_TRANSFORMER_PROOF"
+SOURCE_DATE_EPOCH_DEFAULT = 0
+DEFAULT_WORK_DIR = pathlib.Path(os.environ.get("ZKAI_JSTPROVE_SHAPE_WORK_DIR", "/tmp/zkai-jstprove-shape-probe"))
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/zkai-jstprove-shape-probe"

(S108)